Gemfile

```diff
@@ -3,6 +3,8 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'mysql2'
+gem 'pg'
+gem 'sqlite3'
```

Comment:
Gemspec are them gem dependencies. While  in the gemfile you put things that are only for dev/test